account-keys/src/account_keys.rs

```diff
@@ -318,6 +332,30 @@ impl AccountKey {
         }
     }
 
+    /// Attach fog fields to an `AccountKey` instance
```

Comment:
it is a new instance, but it's consuming `self`, how would you prefer this described?

Reply:
While I understand it might be exposing the abstraction.  The signature takes a `self` not a `mut self`.  If one were only to look at the signature and the description it might make one question practices, i.e. is this implementation going into `unsafe` to make the item mutable.
```suggestion
    /// Copy to a new instance using with provided fog fields
```
Not sure I like the "copy to"...  Originally I had "Create a new..." but then it didn't feel clear enough how the keys were handled.